Install guide update for 2022.2 (#12190)
openvinotoolkit/openvino: docs/install_guides/configurations-for-intel-gpu.md

3. **Optional:** Install header files to allow compilation of new code. You can find the header files at [Khronos OpenCL™ API Headers](https://github.com/KhronosGroup/OpenCL-Headers.git).

You've completed all required configuration steps to perform inference on processor graphics. Now you are ready to try out the toolkit:

Developing in Python:
   * [Start with TensorFlow models with OpenVINO™](https://docs.openvino.ai/latest/notebooks/101-tensorflow-to-openvino-with-output.html)
   * [Start with ONNX and PyTorch models with OpenVINO™](https://docs.openvino.ai/latest/notebooks/102-pytorch-onnx-to-openvino-with-output.html)
   * [Start with PaddlePaddle models with OpenVINO™](https://docs.openvino.ai/latest/notebooks/103-paddle-onnx-to-openvino-classification-with-output.html)

Developing in C++:
   * [Image Classification Async C++ Sample](@ref openvino_inference_engine_samples_classification_sample_async_README)
   * [Hello Classification C++ Sample](@ref openvino_inference_engine_samples_hello_classification_README)
   * [Hello Reshape SSD C++ Sample](@ref openvino_inference_engine_samples_hello_reshape_ssd_README)

> 

Reviewer: Nice edits here!

4. Click the **Driver** tab to see the driver version. 
   ![](../img/DeviceDriverVersion.PNG)

> You are done updating your device driver and are ready to use your GPU. Now you are ready to try out the toolkit.

Reviewer: These two parts are the same. Can we make it a dedicated section as something like "Next Steps"?

Author: ```suggestion
You are done updating your device driver and are ready to use your GPU.

## What’s Next?
```

@xu-yuan1 maybe something like this? I used "What's Next?" instead since we are using the same pharse in other installation guides.

Reviewer: Sounds good. Maybe we can keep the "Now you are ready to try out the toolkit." intro sentence in the new section. :)

Author: I thought about this too, but won't it sound a little bit like a repetition of "... ready to use your GPU"? What If we change it to be like this:

```suggestion
You are done updating your device driver and are ready to use your GPU.

## What’s Next?

You can try tryout the toolkit with:
```

Reviewer: Sounds good.
